wordpress-mobile/WordPress-Android
# Create clicks block

Fixes #8678
FluxC endpoint needs to be reviewed and merged with this PR - https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/1022

This PR is adding the Clicks block to the Stats screen. The block contains a list of click items. Some of them are expandable. The list should be different on each Day/Week/Month/Year tab. There should be at most 6 top-tier items in the list. If there are more than 6 items, the `View more` button appears which redirects you to the `View all` activity. We're still reusing the old activity for viewing more clicks.

To test:
* Open Stats
* Go to Days/Weeks/Months/Year tab
* You see a Clicks block at the bottom of the screen
* The block contains different data on different tabs
* Some items are expandable (you have to use a high traffic site to see them)
* If there are 6+ items, the `View more` button appears
* Clicking the `View more` button redirects you to the `View all Clicks` activity
